### PR TITLE
Jpff1886

### PR DESCRIPTION
**Pull request by jpffitch**
Fivx for #1886

### CODE REVIEW

File: BUILD.md
Comment (vlazzarini) [inline BUILD.md:1]: why was this deleted?

File: all_string_files
Comment (vlazzarini) [inline all_string_files:1]: why was this deleted?

**Comment by vlazzarini** [comment]
Builds are failing?
The only other thing is that there are a number of deletions, not sure why

**Comment by jpffitch** [comment]
;2~No idea why build,md was removed -- I have been trying to find it.

I noticed all_string-files was deleted and i trie to preserve copy. 
Not sure where I put it.

**Comment by kunstmusik** [comment]
I think something's gone off with this PR, csound_orc.lex seems to have gotten replaced with a different file. Also, unit tests for web audio were removed.

**Comment by jpffitch** [comment]
I did add words to the PR to warn that something is very wrong.  But I 
could not find out how to delete the PR

**Comment by jpffitch** [comment]
I think I need more instructions for creating and submitting a PR

**Comment by vlazzarini** [comment]
What I can do is this. I can set up the branch, submit the PR and you can check the branch out, do the changes and commit them.
Would that be helpful?

Prof. Victor Lazzarini
Maynooth University
Ireland

On 17 Jun 2024, at 21:49, John ffitch ***@***.***> wrote:

﻿
*Warning*

This email originated from outside of Maynooth University's Mail System. Do not reply, click links or open attachments unless you recognise the sender and know the content is safe.

I think I need more instructions for creating and submitting a PR
